.travis.yml

```diff
@@ -2,12 +2,14 @@ language: php
 
 php:
   - 5.6
-  - 7.0
+  - 7.0.19
```

Comment:
Yes, that was a bad idea, I just wanted to try if we can force latest PHP version. CircleCI config is updated now.